src/app/utils/number-utils.ts

```diff
@@ -0,0 +1,14 @@
+import BigNumber from 'bignumber.js'
+
+const GWEI_DENOMINATION = BigNumber(10).pow(9)
+
+export const getGasPrice = ({ fee = '', gasUsed = '' }): string | null => {
+  if (!fee || !gasUsed) {
+    return null
+  }
+
+  return BigNumber(fee).div(BigNumber(gasUsed)).toLocaleString()
```

Comment:
Can `toLocaleString` brake `getPreciseNumberFormat`?

Reply:
Maybe. It's odd that it even works
- `toLocaleString` comes from Object in BigNumber's prototype chain
- Object.toLocaleString natively calls `this.toString`
- BigNumber.toString sometimes returns exponential format (1e+28, 1.234e-35).

toFixed is the safest option
https://github.com/oasisprotocol/explorer/blob/ed691bb3a65963b1c8345dd279454cab56e70b72/src/app/utils/helpers.ts#L113-L119

Reply:
Yes, `toFixed` looks good. Thanks 👍 

It's true, that `toString` returns exponent format, but looks like `Intl.NumberFormat` correctly parsed it.